suites/nautilus/rgw/tier-1_rgw_ssl_test-upgrade-4-to-latest.yaml

```diff
@@ -0,0 +1,255 @@
+#
+# Objective: Test rgw with ssl configured using ceph-ansible
```

Comment:
Could we add `upgrade` in the objective?

Reply:
@psathyan sure added